src/messages/MessageBuilder.cpp

```diff
@@ -223,6 +223,96 @@ std::pair<MessagePtr, MessagePtr> makeAutomodMessage(
     return std::make_pair(message1, message2);
 }
 
+std::pair<MessagePtr, MessagePtr> makeLowTrustUserMessage(
+    const LowTrustUser_NewMessageAction &action)
+{
+    MessageBuilder builder, builder2;
```

Comment:
We would preferably not be building multiple messages

Reply:
I am unsure about what I have to fix here, I only copied Automod messages and modified it